app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -166,4 +153,15 @@ class ProfileFragment : Fragment() {
         setHasOptionsMenu(true)
         super.onResume()
     }
+
+    private fun redirectToLogin() {
+        val intent = Intent(activity, AuthActivity::class.java)
+        intent.putExtra("textSnackbar", "You need to log in first!")
```

Comment:
Change intent extra name to `snackbar_message` and make it a constant

Reply:
> make it a constant

means I have to store and extract the strings

Reply:
Yes